src/perform_step/exponential_rk_perform_step.jl

```diff
@@ -14,7 +14,11 @@ function perform_step!(integrator, cache::LawsonEulerConstantCache, repeat_step=
   @unpack t,dt,uprev,u,f,p = integrator
   rtmp = integrator.fsalfirst
   A = f.f1
-  @muladd u = expm(dt*A)*(uprev + dt*rtmp)
+  if integrator.alg.krylov
+    @muladd u = expmv(dt, A, uprev + dt*rtmp; tol=integrator.opts.reltol)
+  else
+    @muladd u = expm(dt*A)*(uprev + dt*rtmp)
```

Comment:
Should use the cached `expA` here? Looks like that's an error from the original code.